pydis_core/utils/scheduling.py

```diff
@@ -219,6 +219,7 @@ def create_task(
     *,
     suppressed_exceptions: tuple[type[Exception], ...] = (),
     event_loop: typing.Optional[asyncio.AbstractEventLoop] = None,
+    done_callbacks: tuple[typing.Callable[[asyncio.Task], any], ...] = (),
```

Comment:
```suggestion
    done_callbacks: tuple[typing.Callable[[asyncio.Task, ...], any], ...] = (),
```
I _think_ this is how you'd type this, since the callable _could_ be a partial that sets some args/kwargs before passing it into this func.

Reply:
3a793d0